### DISCUSSION TITLE
Install Guide

### DISCUSSION THREAD

**Discussion by afterbit01** [primefaces/community#2255] [PrimeFaces Templates]
Hello, 
I have a web JSF website with PF 13. 
I buyed verona layout. 
Now I need a guide to install it .

(i searched similar discussions and also on google, but without success)

thanks! 
Roberto.

**Comment by melloware** [comment]
@afterbit01 isn't that here: https://www.primefaces.org/verona/documentation.xhtml;jsessionid=node014ab8scxz22k2144etwvak6f6z29487061.node0

You basically start with the verona running project that comes with your download and pull out the layout and stuff you need.

**Reply by afterbit01** [reply]
Hi!
yes, I saw that page before adding this discussion, but I already have a Maven project and I need to install the Verona theme on it..
I would like a guide that shows which files need to be copied into my project,
I'll try to integrate the .jar and so on..

**Comment by tandraschko** [comment]
i will add docs for integrating it into a existing maven application for the next release

**Comment by ritonglue** [comment]
This is a quick guide:

- update primefaces version in pom.xml
- copy org.primefaces.verona.GuestPreferences in your project
- copy org.primefaces.verona.component package in your project
- copy primefaces-verona.taglib.xml
- modify in web.xml jakarta.faces.FACELETS_LIBRARIES and primefaces.THEME (look at verona web.xml)
- copy config.xhtml
- optional : copy the error pages (notfound.xhtml, access.xhtml, error.xhtml)
- copy faces-config.xml content in your faces-config.xml
- copy menu.xhtml (and adapt to your project)
- copy footer.xhtml (and adapt to your project)
- copy topbar.xhtml (and adapt to your project)
- copy template.xhtml (and adapt to your project)
- copy webapp/resources content to your webapp/resources
- run the sass command : `sass --update src/main/webapp/resources:src/main/webapp/resources --no-source-map`

You don't need config.xhtml or footer.xhtml if you remove them from template.xhtml

The error pages are of interest if you use them in your web.xml

Have fun !

**Comment by tandraschko** [comment]
Thanks, i will use this as base!
